app/screens/Course/text.ts

```diff
@@ -0,0 +1 @@
+export const text = 'National bank exchange rate';
```

Comment:
Name it correctly
I don't understand what this `text` variable does.
Also, use UPPERCASE for CONSTANTS